snuba/datasets/dataset.py

```diff
@@ -118,6 +118,14 @@ def get_all_storages(self) -> Sequence[Storage]:
         """
         return self.__storages
 
+    def get_writable_storage(self) -> Optional[WritableTableStorage]:
+        """
+        Temporarily support getting the writable storage from a dataset.
+        Once consumers/replacers no longer reference datasets, this can be removed
+        and datasets can have more than one writable storage.
+        """
+        return self.__writable_storage
+
     def get_table_writer(self) -> Optional[TableWriter]:
```

Comment:
Do you still need this one ?

Reply:
It's not needed (i'd remove this and enforce_table_writer together as a follow up)